.github/workflows/main_ci.yml

```diff
@@ -24,10 +24,21 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install --upgrade numpy mypy
+          python -m pip install --upgrade numpy mypy cmake pytest pybind11
+      - name: Install pykokkos-base
```

Comment:
Looks like this compilation completes in about 3 minutes or less in the Linux GitHub actions here.

Reply:
That's reasonable